config/default_records/plate_purposes/009_bioscan_purposes.yml

```diff
@@ -12,11 +12,18 @@ LILYS-96 Stock:
   type: PlatePurpose::Input
   stock_plate: true
   cherrypickable_target: false
-# This is the lysed material plate, ready for the Library prep pipeline. These plates
-# are either prepared by the faculty team manually, or are created on the Beckman robot
-# from the LILYS-96 Stock parent.
-# These plates include controls.
+# This is the lysed material plate, ready for the Library prep pipeline.
+# This version of the Lysate plate is created on the Beckman robot from
+# the LILYS-96 Stock parent.
+# These plates include 2 added controls.
 LBSN-96 Lysate:
-  type: PlatePurpose::AdditionalInput
```

Comment:
Explicitly showing `input_plate: false ` would be useful to see the contrast with the `Input` version of this purpose.

Reply:
yes, added